planner/core/exhaust_physical_plans.go

```diff
@@ -2677,6 +2677,11 @@ func (p *baseLogicalPlan) canPushToCopImpl(storeTp kv.StoreType, considerDual bo
 				if len(path.PartialIndexPaths) > 0 {
 					considerIndexMerge = true
 				}
+				// Global index may be filtered for selected partitions in TiDB, so cannot be pushed down now.
+				// TODO(L-Maple): add cop support for global index.
```

Comment:
Thanks, relates to a previous question I had above.

Reply:
by following your advice, this logic is removed now.